source/extensions/filters/http/jwt_authn/jwks_cache.h

```diff
@@ -72,6 +72,9 @@ class JwksCache {
     virtual JwtCache& getJwtCache() PURE;
   };
 
+  // Lookup provider cache map. The cache only stores Jwks specified in the config.
+  virtual JwksData* getDefault() PURE;
```

Comment:
The comment should be
// If there is only one provider in the config, return the data for that provider. 
// It is only used for checking "failed_status_in_metadata" config for now.

maybe it is better to change the function name to "getSingleProvider"